UnitsNet/UnitDefinitions/HeatFlux.json

```diff
@@ -41,75 +41,75 @@
       ]
     },
     {
-      "SingularName": "BtuPerSquareInchSecond",
-      "PluralName": "BtuPerSquareInchSecond",
+      "SingularName": "BtuPerSecondSquareInch",
+      "PluralName": "BtuPerSecondSquareInch",
       "FromUnitToBaseFunc": "x*1.63533984e6",
       "FromBaseToUnitFunc": "x/1.63533984e6",
       "Localization": [
         {
           "Culture": "en-US",
-          "Abbreviations": [ "BTU/in²/s" ]
+          "Abbreviations": [ "BTU/s·in²" ]
         }
       ]
     },
     {
-      "SingularName": "BtuPerSquareFootSecond",
-      "PluralName": "BtuPerSquareFootSecond",
+      "SingularName": "BtuPerSecondSquareFoot",
+      "PluralName": "BtuPerSecondSquareFoot",
       "FromUnitToBaseFunc": "x*1.13565267e4",
       "FromBaseToUnitFunc": "x/1.13565267e4",
       "Localization": [
         {
           "Culture": "en-US",
-          "Abbreviations": [ "BTU/ft²/s" ]
+          "Abbreviations": [ "BTU/s·ft²" ]
         }
       ]
     },
     {
-      "SingularName": "BtuPerSquareFootMinute",
-      "PluralName": "BtuPerSquareFootMinute",
+      "SingularName": "BtuPerMinuteSquareFoot",
+      "PluralName": "BtuPerMinuteSquareFoot",
       "FromUnitToBaseFunc": "x*1.89275445e2",
       "FromBaseToUnitFunc": "x/1.89275445e2",
       "Localization": [
         {
           "Culture": "en-US",
-          "Abbreviations": [ "BTU/ft²/min" ]
+          "Abbreviations": [ "BTU/min·ft²" ]
         }
       ]
     },
     {
-      "SingularName": "BtuPerSquareFootHour",
-      "PluralName": "BtuPerSquareFootHour",
+      "SingularName": "BtuPerHourSquareFoot",
+      "PluralName": "BtuPerHourSquareFoot",
       "FromUnitToBaseFunc": "x*3.15459075",
       "FromBaseToUnitFunc": "x/3.15459075",
       "Localization": [
         {
           "Culture": "en-US",
-          "Abbreviations": [ "BTU/ft²/hr" ]
+          "Abbreviations": [ "BTU/hr·ft²" ]
         }
       ]
     },
     {
-      "SingularName": "CalPerSquareCentiMeterSecond",
-      "PluralName": "CalPerSquareCentiMeterSecond",
+      "SingularName": "CalPerSecondSquareCentimeter",
+      "PluralName": "CalPerSecondSquareCentimeter",
       "FromUnitToBaseFunc": "x*4.1840e4",
       "FromBaseToUnitFunc": "x/4.1840e4",
       "Prefixes": [ "Kilo" ],
       "Localization": [
         {
           "Culture": "en-US",
-          "Abbreviations": [ "cal/cm²/s" ]
+          "Abbreviations": [ "cal/s·cm²" ]
         }
       ]
     },
     {
-      "SingularName": "KiloCalPerSquareMeterHour",
-      "PluralName": "KiloCalPerSquareMeterHour",
+      "SingularName": "KilocalPerHourSquareMeter",
+      "PluralName": "KilocalPerHourSquareMeter",
       "FromUnitToBaseFunc": "x*1.16222222",
       "FromBaseToUnitFunc": "x/1.16222222",
       "Localization": [
         {
           "Culture": "en-US",
-          "Abbreviations": [ "kcal/m²/hr" ]
+          "Abbreviations": [ "kcal/hr·m²" ]
```

Comment:
You need to add parantheses here and for the above units, like: `kcal/(hr·m²)`

Reply:
I have never seen it written this way. Order of operations should already take care of it, no? https://www.pentairthermal.com/Images/EN-IndustrialHeatTracingUnitConversionTable-AR-H57432_tcm432-26156.pdf

Reply:
Hm, good point. I'm not really sure what are the conventions on this, but I too have seen the form without the parantheses. I thought precedence was not universal for math, but rather chosen and defined per programming language.

https://en.wikipedia.org/wiki/Order_of_operations
Here it seems that division by x is treated just like an multiplication of `1/x`, and if I got that right then `kcal/hr·m²` becomes `kcal*(1/hr)*m²` - which is not correct.

https://math.berkeley.edu/~gbergman/misc/numbers/ord_ops.html
Here it seems there is no universally agreed upon order, so use parantheses instead.

Do you know if there is any litterature backing up the form of not using parantheses and still being unambiguous? I'm not familiar in this territory.